src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -105,10 +105,33 @@ def metadata_call_credentials(metadata_plugin, name=None):
     A CallCredentials object for use in a GRPCCallOptions object.
   """
   if name is None:
-    name = metadata_plugin.__name__
+    name = metadata_plugin.__class__.__name__
   return CallCredentials(
       _low.call_credentials_metadata_plugin(metadata_plugin, name))
 
+def google_call_credentials(credentials):
+  """Construct CallCredentials from google credentials
+
+  Args:
+    credentials: A google credentials object from oauth2client
+
+  Returns:
+    A CallCredentials object for header authentication
+  """
+  return metadata_call_credentials(_auth.GoogleCallCredentials(credentials))
+
+def access_token_call_credentials(access_token):
+  """Construct CallCredentials from a raw access token
+
+  Args:
+    access_token: A raw access token used for authentication
```

Comment:
What's a "raw access token"? A byte string? An instance of some interface or class?